jcstress-core/src/main/java/org/openjdk/jcstress/infra/runners/TestConfig.java

```diff
@@ -244,4 +244,26 @@ public void generateDirectives(PrintWriter pw, Verbosity verbosity) {
         pw.println("]");
         pw.flush();
     }
+
+
+    public String toDetailedTest() {
```

Comment:
The output looks a bit ugly. I think I understand the motivation for printing out more data about the tests, but then it should be more clean. Right now it looks like just an internals dump, which is not a preferred style for jcstress output.

Or, we can just not do it at all, and just print the test names, without these verbose printouts. That would be my preference.

Reply:
But the individual tests are then not recognizable:(  And the individual difference is jsut big, so ugly at the end.
I had an idea to actually group it in oposite way like:
```
- for this combination
  * those tests are run
  * ...
- for this another combination
  * those tests are run
  * ...
 ```
eg:
```
{[actor1, actor2], spinLoopStyle: Thread.onSpinWait(), threads: 2, forkId: 0, maxFootprintMB: 64, compileMode: 0, shClass: (PG 0, CG 0), (PG 0, CG 0), strideSize: 256, strideCount: 40, cpuMap
: null, [-XX:+UseBiasedLocking]}
  org.openjdk.jcstress.samples.api.API_01_Simple
 org.openjdk.jcstress.tests.volatiles.VolatileIncrementAtomicityTest
{[actor1, actor2], spinLoopStyle: Thread.onSpinWait(), threads: 2, forkId: 0, maxFootprintMB: 64, compileMode: 0, shClass: (PG 0, CG 0), (PG 0, CG 0), strideSize: 256, strideCount: 40, cpuMap
: null, [-XX:-UseBiasedLocking]}
  org.openjdk.jcstress.samples.api.API_01_Simple 
  org.openjdk.jcstress.tests.volatiles.VolatileIncrementAtomicityTest
```
But I guess you got it.
But that was a bit more work which I did not want to start without your feedback.

The verbose test lsiting (toDetailedTest) is super useful. I would highly appreciate to merge it, and am willing to later agree in separate PR/jira issue  on better output(s)

Reply:
All right, fine, let's do this.